docs/ReleaseNotes.md

```diff
@@ -1,6 +1,15 @@
 Release Notes for Version 14
 ============================
 
+Build 005
+-------
+Published as version 14.1.3
```

Comment:
If you put this in the release notes, then please update the build.properties and the package.json to 14.1.3 as well.